app/api/items/get_activity_log.go

```diff
@@ -0,0 +1,359 @@
+package items
+
+import (
+	"errors"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:model itemActivityLogResponseRow
+type itemActivityLogResponseRow struct {
+	// required: true
+	At *database.Time `json:"at"`
+	// required: true
+	// enum: result_started,submission,result_validated
+	ActivityType string `json:"activity_type"`
+	// required: true
+	AttemptID int64 `json:"attempt_id,string"`
+	// `answers.id`
+	AnswerID *int64 `json:"answer_id,string,omitempty"`
+	// use this as `{from.asnwer_id}` for pagination
+	// required: true
+	FromAnswerID int64    `json:"from_answer_id,string"`
+	Score        *float32 `json:"score,omitempty"`
+	// required: true
+	Participant struct {
+		// required: true
+		ID int64 `json:"id,string"`
+		// required: true
+		Name string `json:"name"`
+		// required: true
+		// enum: Team,User
+		Type string `json:"type"`
+	} `json:"participant" gorm:"embedded;embedded_prefix:participant__"`
+	User *struct {
+		// required: true
+		ID *int64 `json:"id,string"`
+		// required: true
+		Login string `json:"login"`
+		// Nullable
+		// required: true
+		FirstName *string `json:"first_name"`
+		// Nullable
+		// required: true
+		LastName *string `json:"last_name"`
+	} `json:"user,omitempty" gorm:"embedded;embedded_prefix:user__"`
+	// required: true
+	Item struct {
+		// required: true
+		ID int64 `json:"id,string"`
+		// required: true
+		// enum: Chapter,Task,Course
```

Comment:
There is no reason to exclude "Skill", are they excluded from the request?

Reply:
No, they are not excluded. The swagger documentation was written before we added skills.

Reply:
fixed